lms/djangoapps/course_api/tests/test_views.py

```diff
@@ -78,6 +77,30 @@ def test_as_honor_for_staff(self):
         self.setup_user(self.honor_user)
         self.verify_response(expected_status_code=403, params={'username': self.staff_user.username})
 
+    @SharedModuleStoreTestCase.modifies_courseware
+    def test_filter_by_org(self):
```

Comment:
There is some (limited) code duplication in these tests. I experimented with a mixin, but feel that it adds unnecessary complexity. That said, I'm open to revisiting a mixin if reviewers feel that these tests would stand to benefit from one.